images/cassandra/initialize.sh

```diff
@@ -9,4 +9,4 @@ fi
 
 sed -i 's/REPLICATION_FACTOR/'"${REPLICATION_FACTOR}"'/g' /init.cql
 
-cqlsh -f /init.cql cassandra.default.svc
+cqlsh -f /init.cql cassandra.default.svc.cluster.local
```

Comment:
what about cert authority? I don't think it's set up in the way to trust this name